crates/aptos/src/stake/mod.rs

```diff
@@ -297,3 +303,126 @@ impl CliCommand<TransactionSummary> for CreateStakingContract {
+    fn command_name(&self) -> &'static str {
+        "UnlockVestingRewards"
+    }
+
+    async fn execute(mut self) -> CliTypedResult<TransactionSummary> {
+        let vesting_contract_address = create_vesting_contract_address(
+            self.admin_address,
+            0,
+            &[]
+        );
+        prompt_yes_with_override(
+            &format!(
+                "Unlocking rewards from vesting contract {}. Confirm?",
+                vesting_contract_address
+            ),
+            self.txn_options.prompt_options,
+        )?;
+
+        self.txn_options
+            .submit_transaction(aptos_stdlib::vesting_unlock_rewards(
+                vesting_contract_address
+            ))
+            .await
+            .map(|inner| inner.into())
+    }
+}
+
+#[derive(Parser)]
+pub struct VestingDistribute {
+    /// Address of the vesting contract's admin.
+    ///
+    /// Defaults to the profile's address if not set.
+    #[clap(long, parse(try_from_str=crate::common::types::load_account_arg))]
+    pub admin_address: AccountAddress,
+
+    #[clap(flatten)]
+    pub(crate) txn_options: TransactionOptions,
+}
+
+#[async_trait]
+impl CliCommand<TransactionSummary> for VestingDistribute {
+    fn command_name(&self) -> &'static str {
+        "VestingDistribute"
+    }
+
+    async fn execute(mut self) -> CliTypedResult<TransactionSummary> {
+        let vesting_contract_address = create_vesting_contract_address(
+            self.admin_address,
+            0,
+            &[]
+        );
+        prompt_yes_with_override(
+            &format!(
+                "Distribute coins for vesting contract {}. Confirm?",
+                vesting_contract_address
+            ),
+            self.txn_options.prompt_options,
+        )?;
+
+        self.txn_options
+            .submit_transaction(aptos_stdlib::vesting_distribute(
+                vesting_contract_address
+            ))
+            .await
+            .map(|inner| inner.into())
+    }
+}
+
+#[derive(Parser)]
+pub struct UnlockVestedCoins {
+    /// Address of the vesting contract's admin.
+    ///
+    /// Defaults to the profile's address if not set.
+    #[clap(long, parse(try_from_str=crate::common::types::load_account_arg))]
+    pub admin_address: AccountAddress,
+
+    #[clap(flatten)]
+    pub(crate) txn_options: TransactionOptions,
+}
+
+#[async_trait]
+impl CliCommand<TransactionSummary> for UnlockVestedCoins {
+    fn command_name(&self) -> &'static str {
+        "UnlockVestedCoins"
+    }
+
+    async fn execute(mut self) -> CliTypedResult<TransactionSummary> {
+        let vesting_contract_address = create_vesting_contract_address(
+            self.admin_address,
+            0,
+            &[]
+        );
+        prompt_yes_with_override(
+            &format!(
+                "Unlock vested coins for vesting contract {}. Confirm?",
+                vesting_contract_address
+            ),
+            self.txn_options.prompt_options,
+        )?;
```

Comment:
I might get rid of these, because it asks you again when you simulate the transaction for confirmation.  But, it's fine for now

Reply:
Deleted them